automation/tests/cloning/mongo-si_test/mongo_si_test.go

```diff
@@ -16,26 +18,28 @@ import (
 
 // A test suite is a collection of related test cases that are grouped together for testing a specific package or functionality.
 // The testify package builds on top of Go's built-in testing package and enhances it with additional features like assertions and test suite management.
-// PostgresqlSingleInstanceTestSuite is a test suite struct that embeds testify's suite.Suite
-type PostgresqlSingleInstanceTestSuite struct {
+// MongoCloningSingleInstanceTestSuite is a test suite struct that embeds testify's suite.Suite
+type MongoCloningSingleInstanceTestSuite struct {
 	suite.Suite
 	ctx               context.Context
 	setupTypes        *util.SetupTypes
 	v1alpha1ClientSet *clientsetv1alpha1.V1alpha1Client
 	clientset         *kubernetes.Clientset
+	tms               util.TestSuiteManager
 }
 
 // SetupSuite is called once before running the tests in the suite
-func (suite *PostgresqlSingleInstanceTestSuite) SetupSuite() {
+func (suite *MongoCloningSingleInstanceTestSuite) SetupSuite() {
 	var err error
 	var config *rest.Config
 
 	var ctx context.Context
 	var v1alpha1ClientSet *clientsetv1alpha1.V1alpha1Client
 	var clientset *kubernetes.Clientset
+	var tms util.TestSuiteManager
 
 	// Setup logger and context
-	logger, err := util.SetupLogger("./pg-si_test.log")
+	logger, err := util.SetupLogger(fmt.Sprintf("%s/mongo-cloning-si_test.log", automation.CLONING_LOG_PATH), "mongo-cloning-si: ")
 	if err != nil {
 		suite.T().FailNow()
```

Comment:
Might be useful to log before failing.

Reply:
If the logger is unable to be created, I won't be able to log to the log file. In the case, should I log to the console?